src/controllers/featureFlags/featureFlags.ts

```diff
@@ -0,0 +1,25 @@
+import { FeatureFlags, featureFlags } from '../../consts/featureFlags'
+import EventEmitter from '../eventEmitter/eventEmitter'
+
+/**
+ * Responsible for managing (enable/disable) feature flags within the app. The
+ * long-term vision for this is to support dynamic management, currently it
+ * enables or disables features only at runtime. Can be useful for feature
+ * toggling, A/B testing, and gradual feature roll-outs.
+ */
+export class FeatureFlagsController extends EventEmitter {
+  flags = { ...featureFlags }
```

Comment:
I guess this should be private, as we have a setter function and the same time we can mutate its value outside of the class

Reply:
Done just now